my-sms/spec/controllers/courses_controller_spec.rb

```diff
@@ -116,6 +116,26 @@
         is_expected.to be_success
         is_expected.to render_template('edit')
       end
+
+      it 'should propagate model errors to an error flash' do
+        expect { subject }.to change { flash[:error] }.from(nil).to(['End date must be after start date'])
+      end
+    end
+  end
+
+  describe 'DELETE #destroy' do
+    let!(:course) { create(:course) }
+    subject { delete :destroy, { id: course.to_param } }
+
+    # before { course }
```

Comment:
can ✂️ the commented code

Reply:
Fixed in 0a81a79d2c2fd14fc24a1a1e615b28555bb4fa17